src/command.ts

```diff
@@ -13,9 +13,13 @@ import {
 
 export type MaybeArray<T> = T | T[];
 
+const isAdmin = (ctx: Context) =>
+  ctx.getAuthor().then((author) => ["administrator", "creator"].includes(author.status));
+
 export class Command<C extends Context = Context> extends Composer<C> {
   #scopes: BotCommandScope[] = [];
   #languages: Map<string, { name: string; description: string }> = new Map();
+  #middleware: Array<Middleware<C>>;
```

Comment:
This will be transpired to using Weakmaps for Node 12 AFAIK, so it's usually preferred to use `private`, which is good enough for TS people.

Reply:
I see.
Okay, I'll refactor